### PR TITLE
Nerfs sec budget

### PR DESCRIPTION
**Pull request by Bibby0110**
# General Documentation

### Intent of your Pull Request
Due to the fact that the economy / every subsystem doesn't understand that 5 minutes isn't 2.74 seconds the Sec budget (and every budget) was getting thousands of credits every few minutes. This is a bandaid fix but I don't know subsystem code
Removes the 100k sec credits 30 minutes in.

### Why is this change good for the game?

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
no

### What general grouping does this PR fall under? 
Balance

### Are there any aspects of the PR that you would like us not to mention on the Wiki?
no

### If there are any numerical values involved in your PR that will be relevant to a player, please note them here. 
Sec budget gets max 750 credits per cycle

# Changelog

:cl:  
tweak: Sec budget has been reduced by 75%
/:cl:

### CODE REVIEW

**Comment by jcatjmeow** [comment]
I agree with this change, but you've listed no reasons. Why? Maybe take like 4 seconds to properly fill out the template xoxo

**Comment by Bibby0110** [comment]
done

**Comment by redmoogle** [comment]
Can you fill out the PR template a little better. K thx chief
Mainly this
```
Briefly describe your PR and the impacts of it, in layman's terms.

no
```

**Comment by Bibby0110** [comment]
> Can you fill out the PR template a little better. K thx chief
> Mainly this
> 
> ```
> Briefly describe your PR and the impacts of it, in layman's terms.
> 
> no
> ```

It’s described above so please read
